packages/cysync-core/src/pages/MainApp/Portfolio.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Why do we need to disable these?

Reply:
As, currently there is no handlers to handle `setStates` of the states, I have added to remove the error while committing the code.